crypto-test/.jvm/src/test/scala/org/bitcoins/crypto/BouncyCastleBIP340Test.scala

```diff
@@ -32,9 +32,10 @@ class BouncyCastleBIP340Test extends BitcoinSCryptoTest {
       sig: SchnorrDigitalSignature,
       expectedResult: Boolean,
       comment: String): Assertion = {
-    val secpResult = pubKey.verify(msg, sig)
+    val secpResult = Try(pubKey.verify(msg, sig)).getOrElse(false)
```

Comment:
First one isn't needed (invalid additions and such are handled internally by secp which gives a return code of 0 instead of 1), second one is now needed because pubkey combine is used for addition which can fail for invalid additions.